src/Google/GlobalSiteTag.php

```diff
@@ -91,10 +124,10 @@ public function activate_global_site_tag( string $ads_conversion_id ) {
 		if ( $this->gtag_js->is_adding_framework() ) {
 			add_filter(
 				'woocommerce_gtag_snippet',
-				function( $gtag_snippet ) use ( $ads_conversion_id ) {
+				function ( $gtag_snippet ) use ( $ads_conversion_id ) {
 					return preg_replace(
 						'~(\s)</script>~',
-						"\tgtag('config', '" . $ads_conversion_id . "');\n$1</script>",
+						"\tgtag('config', '" . $ads_conversion_id . "', { 'groups': 'GLA', 'send_page_view': 'false' });\n$1</script>",
```

Comment:
According to the [docs](https://support.google.com/analytics/answer/11403294?hl=en) the `send_page_view` should be set to just a boolean false.

Reply:
Updated.